src/preprocess/query_generator.py

```diff
@@ -109,6 +113,79 @@ def generate_delete_tracker_rules(trackers, data_elements, org_units, org_unit_d
+    f.write("DELETE FROM  interpretation_comments where interpretationid in (select interpretationid from interpretation where organisationunitid in (select * from orgUnitsToDelete));\n");
+    f.write("DELETE FROM  intepretation_likedby where interpretationid in (select interpretationid from interpretation where organisationunitid in (select * from orgUnitsToDelete));\n");
+    f.write("DELETE FROM  interpretationuseraccesses where interpretationid in (select interpretationid from interpretation where organisationunitid in (select * from orgUnitsToDelete));\n")
+    f.write("DELETE FROM  interpretation where organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM  lockexception where organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM  mapview_organisationunits where organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM  organisationunitattributevalues where organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM  program_organisationunits where organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM  programinstanceaudit where programinstanceid in (select programinstanceid from programinstance where organisationunitid in (select * from orgUnitsToDelete));\n")
+    f.write("DELETE FROM  programinstance where organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM  programmessage where organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM  reporttable_organisationunits where organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM trackedentityprogramowner WHERE organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM  trackedentityinstance where organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM  userdatavieworgunits where organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM  usermembership where organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM  userteisearchorgunits where organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM  validationresult where organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM completedatasetregistration where sourceid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM configuration WHERE selfregistrationorgunit in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM minmaxdataelement WHERE sourceid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM visualization_organisationunits WHERE organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DELETE FROM organisationunit WHERE organisationunitid in (select * from orgUnitsToDelete);\n")
+    f.write("DROP MATERIALIZED VIEW if exists orgUnitsToDelete;" + "\n")
+
```

Comment:
Does this work to avoid so many `f.write`?

f.write("""
  SQL1;
  SQL2;
  ...
""")

Reply:
done